tool/options.go

```diff
@@ -73,6 +74,13 @@ func ProcessFlags() {
 		false,
 		"Use a legacy leader election method for controller manager instead of the lease API.",
 	)
+
+	flag.StringVar(
```

Comment:
This value needs to be used in `manager.Options`.

Reply:
Thanks. I was wondering why the kind test failed at pod not ready. You saved me a lot of time!